### ISSUE TITLE
Results from tensorboard during training is better

### ISSUE THREAD

**Issue by leonardoaraujosantos** [leonardoaraujosantos/LearnSegmentation#2]
## Introduction
The results from tensorboard are better than the prediction time.

### Loss from tensorboard
Roughly 0.7 after 1 day of training
![loss](https://user-images.githubusercontent.com/898383/27611246-604e3b48-5b89-11e7-9afb-fe6aa776c491.png)

### Result on Tensorboard
![bedontensorboard](https://user-images.githubusercontent.com/898383/27611287-88d5d62a-5b89-11e7-98e8-139f5bd49842.png)

### Result from Prediction Notebook
[Tensorflow notebook](https://github.com/leonardoaraujosantos/LearnSegmentation/blob/master/src/notebooks/Tensorflow_Segmentation.ipynb)
![resultbad](https://user-images.githubusercontent.com/898383/27611448-1f8d37b6-5b8a-11e7-8bc2-5ce7cf94887a.png)

**Comment by leonardoaraujosantos** [comment]
### Solution
The problem was that I was initializing the model without indicating to the batchnorm that I was doing inference not training
``` python
# Build model
if segmentation_type == 'fcn':
    segmentation_model = models.FullyConvolutionalNetworks(training_mode = False)
elif mode.lower() == 'deconvnet':
    segmentation_model = models.FullyConvolutionalNetworks(training_mode = False)
elif mode.lower() == 'segnet':
    segmentation_model = models.FullyConvolutionalNetworks(training_mode = False)
else:
    segmentation_model = models.FullyConvolutionalNetworks(training_mode = False)
```

#### Result after
![resultgood](https://user-images.githubusercontent.com/898383/27611564-97704624-5b8a-11e7-968c-2dd4db9eed48.png)
